mtp_cashbook/apps/mtp_auth/exceptions.py

```diff
@@ -0,0 +1,6 @@
+class ConnectionError(Exception):
+    """
+    Exception that should be raised in case of connection
+    problems with the api server.
+    """
+    pass
```

Comment:
Yes I knew that you might have not liked this.
It's not confusing for me as I wanted to make explicit that you need to catch **this** exception and understand what it means for you as dev when you do it.
If it's confusing for you, I might just use the requests one or the python3 one, doesn't really change anything for me really.